### PR TITLE
M3-5495: UI refinements for Volume Create

### PR DESCRIPTION
**Pull request by tiffwong**
## Description

This PR adds contextual help, rearranges the Size fields, makes the Linode dropdown clearable, and removes the Tags section. The width of the fields have also been reduced.

## How to test

There are several type changes to components that are used frequently in Cloud Manager so check the other forms for regressions.

### CODE REVIEW

**Review by HanaXu — COMMENTED** [review]
Should we move the label icon next to the input field (as seen in the label)?

image

**Review by HanaXu — CHANGES_REQUESTED** [review]
There's a bug with the region not clearing if we select a region, then select a linode, and clear the selected linode. However, if we skip selecting a region and select a linode to prefill region, then clear the selected linode, the region clears.

There's also a whitespace issue with the dropdown expanded

https://user-images.githubusercontent.com/14323019/158682531-5d191987-b92c-459d-b527-1a30e221288d.mov

**Review by DevDW — APPROVED** [review]
LGTM 👌

**Review by bnussman — COMMENTED** [review]
In progress of reviewing but have a few comments. 

It would be nice if this paper took up the whole width of the container. This patterns is being used just about everywhere now (Linode Create, Images Create, ...)
Screen Shot 2022-03-18 at 10 15 45 AM

Not really a change but I hope we can find some consistency on this button placement. In some places it's in the paper, in other places it's outside...
Screen Shot 2022-03-18 at 10 19 13 AM

File: packages/manager/src/components/EnhancedSelect/Select.tsx [hunk 1]
```diff
@@ -102,7 +102,7 @@ export interface BaseSelectProps
   /**
    * We require label for accessibility purpose
    */
-  label: string;
+  label: string | JSX.Element;
```
Comment (HanaXu) [inline packages/manager/src/components/EnhancedSelect/Select.tsx:105]: Do we still need this change?

File: packages/manager/src/features/Volumes/VolumeCreate/CreateVolumeForm.tsx [hunk 2]
```diff
@@ -240,6 +263,18 @@ const CreateVolumeForm: React.FC<CombinedProps> = (props) => {
...
 
+        const handleLinodeChange = (linode: Linode | null) => {
+          if (linode !== null) {
+            setFieldValue('linode_id', linode.id);
+            setFieldValue('region', linode.region);
+            setLinodeId(linode.id);
+          } else {
+            // If the LinodeSelect is cleared, reset the values for Region and Config
+            setFieldValue('linode_id', -1);
+            setLinodeId(initialValueDefaultId);
```
Comment (HanaXu) [inline packages/manager/src/features/Volumes/VolumeCreate/CreateVolumeForm.tsx:274]:
```suggestion
            setFieldValue('linode_id', initialValueDefaultId);
            setLinodeId(initialValueDefaultId);
```

**Review by HanaXu — APPROVED** [review]
LGTM, left some minor clean up comments 🚢

**Pull request merged**